packages/runtime-tags/src/common/errors.ts

```
@@ -22,6 +24,23 @@ export function _assert_hoist(value: unknown) {
   }
 }
 
+export function _assert_init(scope: Scope, accessor: string) {
+  if (MARKO_DEBUG && (scope[AccessorProp.Creating] || !(accessor in scope))) {
+    try {
+      // @ts-expect-error create a browser uninitialized variable error, and then update the message.
+      __UNINITIALIZED__;
```

Biome (2.4.6)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)